IM_calculation/Advanced_IM/runlibs_2d.py

```diff
@@ -44,6 +53,15 @@ def parse_args():
     return args
 
 
+def datetime_to_file(t_value, t_type, out_dir):
```

Comment:
Should be clear that this is the string not the enum.

Reply:
this verify script is already importing functions from runlib2d.py . moving this to other places may increases the dependancies, instead of reducing it.

Reply:
Thoughts on moving the verify script into IM_calc rather than having it part of workflow? It doesn't seem to have much slurm dependancies? @jpa198

Reply:
my original reason for putting the verify script in workflow is because all other verification codes are in there as well (emod3d, hf, bb). and since Adv_IM is part of the automated workflow, i thought this makes it more consistent.
However, moving it from workflow to IM_calc also removes one dependency/connection.

Reply:
That makes sense for hf/bb because their scripts (arguably incorrectly) are there too. 
EMOD is a different question entirely - but all the support scripts for emod are in workflow too.

Reply:
Yeah, that should be in IM_calc rather than slurm_workflow